hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -91,6 +94,8 @@ public class HandleWorkflow {
     private static final Logger logger = LogManager.getLogger(HandleWorkflow.class);
 
     public static final String ALERT_MESSAGE = "Possibly CATASTROPHIC failure";
+    // Temporary flag to control stream mode during transition to block streams
+    public static StreamMode STREAM_MODE = RECORDS;
```

Comment:
we should not be using this static variable anymore. I have added TODO in all the places this should be used. We should use `StreamMode` directly from config.

Reply:
Yes, this was by mistake. Thanks @Neeharika-Sompalli !

Reply:
addressed, thank you!